src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,21 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
 
-@_delegate_statsfuns Normal norm μ σ
-
-gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
```

Comment:
probably a stupid question but why remove `gradlogpdf`?

Reply:
I couldn't find anywhere it was being used or tested. I suspect it was an artifact, but I could be wrong...

Reply:
A quick search highlighted it is used and tested quite a lot. Better idea would be to let it and document it better